thrift/thrift0.13/src/main/java/com/linecorp/armeria/internal/server/thrift/ThriftDescriptiveTypeInfoProvider.java

```diff
@@ -164,6 +169,21 @@ private static FieldRequirement convertRequirement(byte value) {
         }
     }
 
+    static TypeSignature toTypeSignature(Class<?> parentType, FieldMetaData fieldMetaData,
+                                         FieldValueMetaData fieldValueMetaData) {
+        if (!fieldValueMetaData.isTypedef() && !hasTypeDef(fieldValueMetaData)) {
+            return toTypeSignature(fieldValueMetaData);
+        }
+
+        try {
+            final Field field = parentType.getField(fieldMetaData.fieldName);
+            return toTypeSignature(field.getGenericType());
+        } catch (NoSuchFieldException e) {
+            // Ignore exception.
```

Comment:
Question) Is there a particular exception that we wanted to ignore? If not, I wonder if debugging would help with troubleshooting potential issues

Reply:
I thought `NoSuchFieldException` won't be raised and just added the catch clause because it's a checked exception.
So you mean we need to at least leave a debug log?

Reply:
Oh I see, I guess there's no need to leave a log if we're sure this won't happen 😅